tests/testthat/api.github.com/organizations-5377e8.R

```diff
@@ -0,0 +1,47 @@
+structure(list(url = "https://api.github.com/organizations?since=1",
+    status_code = 502L, headers = structure(list(date = "Fri, 16 Oct 2020 09:33:06 GMT",
```

Comment:
Since you have `status_code = 502L` and your code calls `stop_for_status()`, you don't really need the rest of the response: you can probably just delete `headers` and `content` from the object, or replace them with `list()` or `NULL` or something.

Reply:
Good point. I created the mock by tweaking as little as possible but it is pointlessly big.